app/src/main/java/com/sun/spoonacular/data/model/Recipe.kt

```diff
@@ -0,0 +1,15 @@
+package com.sun.spoonacular.data.model
+
+import android.os.Parcelable
+import com.google.gson.annotations.Expose
+import com.google.gson.annotations.SerializedName
+import kotlinx.android.parcel.Parcelize
+
+@Parcelize
+data class Recipe(
+    @SerializedName("id") @Expose val id: Int?,
```

Comment:
```suggestion
    @SerializedName("id") 
    @Expose 
    val id: Int?,
```
break line ra cho dễ nhìn nhé